### ISSUE TITLE
`tiden init` should refuse if current folder is not empty

### ISSUE THREAD

**Issue by mikabytes** [tidenjs/tiden#21]
**Describe the bug**
Running init on a folder with files already in it, tiden still sets upp the new files. This has caused me to accidentally overwrite files. It is also annoying when accidentally doing this in your home directory!

**To Reproduce**
```
mkdir `/tmp/test` ; cd /tmp/test
touch "hello.txt"
tiden init test
```

It creates a bunch of files.

**Expected behavior**
It should refuse, clearly stating that init can only be run on empty directories

**Screenshots**

**Additional context**

**Comment by Seke1412** [comment]
I'm working on it. Can you help assign to me man @mikabytes

**Comment by Seke1412** [comment]
I'm giving it a try, please take a look on it @mikabytes man
https://github.com/tidenjs/tiden/pull/29/files

**Comment by mikabytes** [comment]
Fixed in #29
